lib/gvariant.nix

```diff
@@ -0,0 +1,181 @@
+# This file is taken from https://github.com/nix-community/home-manager
```

Comment:
Where should I put the test? Should I add `lib/tests/gvariant.nix`?

Reply:
If you copy https://github.com/nix-community/home-manager/blob/5db22bce05c776057fdb289da17f6c12049c4624/tests/lib/types/gvariant-merge.nix (replacing `home.file` + `nmt.scripts` by direct assertions, it could probably go to to `lib/tests/modules/gvariant.nix`.